docs/GroupDefs.hpp

```diff
@@ -833,6 +833,17 @@ then typically look as follows:
  * inside the debugger.
  */
 
+/*!
+ * \defgroup TimeDependentCoordMapsGroup  Time-dependent Coordinate Maps
```

Comment:
I wonder if this would be better off name `Coordinate Maps, Time-dependent` so that it will be next to the time-independent coordinate maps in the documentation. Any thoughts from others?